src/modules/auth/auth.service.ts

```diff
@@ -158,13 +166,14 @@ export class AuthService {
         from: 'milkav06062003@gmail.com',
         html: `<h1>Change password</h1><p>If you want to change password go to:</p><a href="${url}">${url}</a>`,
       });
-
-      return true;
     }
-    return false;
+    return true;
```

Comment:
so if user don't exis in db it's will return true anyway, does it correct?

Reply:
Yes, it is for FE message. Is user sign up with Google Sign Up or no